server.py

```diff
@@ -15,4 +15,6 @@ def splash_page():
 
 
 # on start
+maxActiveGames = 10
```

Comment:
These would be neat to have in a config file and use the `configparser` module to read them in